tests/route/test_default_route.py

```diff
@@ -17,17 +18,11 @@
 
 logger = logging.getLogger(__name__)
 
-def get_upstream_neigh_type(topo):
-    if 't0' in topo or 'dualtor' in topo:
-        return 'T1'
-    elif 't1' in topo:
-        return 'T2'
-    elif 't2' in topo:
-        return 'T3'
-    elif 'm0' in topo:
-        return 'M1'
-    else:
-        return None
+def get_upstream_neigh_type(topo_type):
+    if topo_type in UPSTREAM_NEIGHBOR_MAP:
+        return UPSTREAM_NEIGHBOR_MAP[topo_type].upper()
```

Comment:
Good refactor! We can consider move this method to common util in future

Reply:
Fixed